content/run.md

```diff
@@ -148,6 +148,101 @@ For dedicated single-cell simulation in order to compare with site-level observa
 
 There are dedicated tools and workflow for running single-cell simulations over Norwegian ecological observation sites using CLM and CLM-FATES, please check [NorESM_LandSites_Platform](https://github.com/NorESMhub/NorESM_LandSites_Platform) and follow the instructions there.
 
+## Run a single cell case by creating surface data
+CLM also supports running single-point simulations by creating single point surface data using python [script](https://github.com/ESCOMP/CTSM/blob/master/python/ctsm/subset_data.py) from NCAR. Note that you need to change the global input data directory variable inside the script (standard global CTSM dataset) as below:
+`dir_inputdata='/cluster/shared/noresm/inputdata/'` (few lines after region.create-tag()).
```

Comment:
Looks like they've rewritten the whole script and has removed some of the args. E.g. `dir_inputdata` doesn't exist anymore. Can you test with the latest version on ESCOMP `master`?